src/frontend/src/components/common/Table/views/useTableStandardView.tsx

```diff
@@ -39,7 +40,7 @@ export const UseTableStandardView = (props: UseTableStandardViewProps) => {
                     >
                       {!props.readOnly ? (
                         <InputText
-                          type="text"
+                          type={TextInputType.INPUTTEXT}
```

Comment:
={column.type ? column.type : TextInputType.INPUTTEXT} 
this logic should still be there but we can fix it at another time.